geotrek/infrastructure/tests/test_views.py

```diff
@@ -52,7 +52,7 @@ def get_expected_json_attrs(self):
 
     def get_expected_datatables_attrs(self):
         return {
-            'cities': '[]',
+            'cities': '',
```

Comment:
Maybe add some cities around the infrastructure geom to test list result ? See `CityFactory`

Reply:
Need to add tests to cover lines requested by codecov